internal/cmd/skupper/link/link.go

```diff
@@ -41,16 +41,16 @@ output needs to be applied in the site in which we want to create the link.`,
 
 	cmd := common.ConfigureCobraCommand(configuredPlatform, cmdLinkGenerateDesc, kubeCommand, nonKubeCommand)
 	cmdFlags := common.CommandLinkGenerateFlags{}
-	cmd.Flags().StringVar(&cmdFlags.TlsCredentials, common.FlagNameTlsCredentials, "", common.FlagDescTlsCredentials)
-	cmd.Flags().StringVar(&cmdFlags.Cost, common.FlagNameCost, "1", common.FlagDescCost)
-	cmd.Flags().StringVarP(&cmdFlags.Output, common.FlagNameOutput, "o", "yaml", common.FlagDescOutput)
-	cmd.Flags().BoolVar(&cmdFlags.GenerateCredential, common.FlagNameGenerateCredential, true, common.FlagDescGenerateCredential)
-	cmd.Flags().DurationVar(&cmdFlags.Timeout, common.FlagNameTimeout, 60*time.Second, common.FlagDescTimeout)
-
+	if configuredPlatform == common.PlatformKubernetes {
```

Comment:
I believe we need two filtering arguments, when using non-kubernetes platforms.
They're both optional and if not provided, it works as is now.

But we could help filtering static links by: `name` (router access name) or `host` (endpoint host).

Reply:
done, let me know if that works for you.